docs/src/guides/gatsby/deploy/deploy.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Deploy Drupal 9"
+sidebarTitle: "Deploy"
+weight: -80
+toc: false
+description: |
+    Now that your site is ready, push it to Platform.sh and import your data.
+---
+
+## Deployment
+
+{{< guides/deployment >}}
+
+## Additional changes
+
+For a standard Gatsby site, or for the example starter blog example, you have all of the configuration necessary to deploy your project. Below are a few modifications that may help you to develop your site more efficiently going forward.
```

Comment:
Need a bit more context here as a reader. Does "standard Gatsby site" mean one with no Gatsby templates? What is the starter blog and where do I get it?